src/ui/Loghandling/BinLogParser.cpp

```diff
@@ -332,14 +349,32 @@ bool BinLogParser::parseDataByDescriptor(QList<NameValuePair> &NameValuePairList
         }
         else if (typeCode == 'd')
         {
+            // we want to read a double -> 8byte -> DoublePrecision
+            packetstream.setFloatingPointPrecision(QDataStream::DoublePrecision);
             double val;
             packetstream >> val;
-            if (qIsInf(val) || qIsNaN(val))
```

Comment:
`qIsNaN` should return true for quiet NaNs.  You could avoid casting to `quint64` by doing the `qIsNaN` and doing the cast in the true case to see just what sort of NaN you're handing.

Reply:
Tried that on double and it did not work - but while reading your comment remembered that the double value was extracted wrong so the `qIsNaN` could not work.
So thanks for that hint - will change it!